plugin/src/main/java/com/craftmend/openaudiomc/generic/redis/RedisService.java

```diff
@@ -59,7 +59,9 @@
 
         // if there are bulk packets waiting, send them
         if (commands.isEmpty()) return;
-        asyncPub.publish(ChannelKey.TRIGGER_BULK_COMMANDS.getRedisChannelName(), new ExecuteBulkCommandsPacket(commands).serialize());
+        ExecuteBulkCommandsPacket packet = new ExecuteBulkCommandsPacket(commands);
+        packet.setSenderUUID(getServiceId());
```

Comment:
the `sendMessage()` below does sets the senderUUID of the packets, but here it doesn't